elf2uf2/elf2uf2.cpp

```diff
@@ -11,49 +11,93 @@
 #include <cstring>
 #include <cstdarg>
 #include <algorithm>
-#include <cstring>
 #include <memory>
 
 #include "elf2uf2.h"
 #include "errors.h"
 
+// Size of a flash sector in bytes
 #define FLASH_SECTOR_ERASE_SIZE 4096u
 
+// Define the page size for UF2 (256 Bytes)
+// These definitions are already present in the header file and should not be redefined here.
+// #define LOG2_PAGE_SIZE 8u
+// #define UF2_PAGE_SIZE (1u << LOG2_PAGE_SIZE)
```

Comment:
These lines can be removed - the definitions had already been removed from this cpp file